src/test/java/com/squareup/kotlinpoet/KotlinPoetTest.kt

```diff
@@ -91,6 +91,36 @@ class KotlinPoetTest {
         |""".trimMargin())
   }
 
+  @Test fun primaryConstructorProperties() {
```

Comment:
We should probably only inline parameters where the initializer is the name of the parameter. Like if it was `val cheese: Cheese = cheese`. That way we’re not risking changing semantics.